python-lib/ner/spacy.py

```diff
@@ -22,22 +22,12 @@
     "nb": "nb_core_news_sm",
 }
 
-def get_spacy_model(language: str):
-    language_model = SPACY_LANGUAGE_MODELS.get(language, None)
-    if language_model is None:
-        raise ValueError(f"The language {language} is not available. \
-                        You can add the language & corresponding model name by editing the code.")
-    try:
-        nlp = spacy.load(language_model, exclude=["tok2vec", "tagger", "parser", "attribute_ruler", "lemmatizer"])
-    except OSError:
-        # Raising ValueError instead of OSError so it shows up at the top of the log
-        raise ValueError(f"Could not find spaCy model for the language {language}. \
-                        Maybe you need to edit the requirements.txt file to enable it.")
```

Comment:
the OS error catch does not make sense anymore since https://github.com/dataiku/dss-plugin-nlp-named-entity-recognition/pull/13